src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -155,8 +155,10 @@ def process_premiere_project(filepath, raven_client, vs_pathmap=None, db=None, c
     lg.info("Project's Vidispine ID: %s" % collection_vsid)
     vsproject = VSCollection(host=cfg.value('vs_host'), port=cfg.value('vs_port'), user=cfg.value('vs_user'),
                              passwd=cfg.value('vs_password'))
-    vsproject.setName(collection_vsid)  #we don't need the actual metadata so don't bother getting it.
-    vsproject.set_metadata({'gnm_project_invalid_media_paths': ''}, mode="add")
+    vsproject.populate(collection_vsid)
+    current_value_of_field = vsproject.get('gnm_project_invalid_media_paths')
+    if current_value_of_field is not None:
```

Comment:
this only checks for the current value being empty - what if it has values, but our update would not change them?

Reply:
That will not happen there. The point of this bit of code it to set this value of the field to a blank string so later bit of code will not break. All it does is set the value of the field to a blank string. Before this change it did it every time. Now it will only do it when the field is not already set to a blank string.

Reply:
with the rewrite of lines 237->246 this is not needed any more